tcmur_cmd_handler.c

```diff
@@ -2308,6 +2316,320 @@ static int handle_inquiry(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
+	};
+
+	cdb_info opcode_start_stop = {
+		.opcode = START_STOP,
+		.cdb_length = 6,
+		.cdb_usage = {START_STOP, 0x01, 0x00, 0x0f, 0xf7, CONTROL_BYTE},     
+	};
+
+	cdb_info opcode_mode_select = {
+		.opcode = MODE_SELECT,
+		.cdb_length = 6,
+		.cdb_usage = {MODE_SELECT, 0x11, 0x00, 0x00, 0xff, CONTROL_BYTE},
+	};
+
+	cdb_info opcode_mode_select_10 = {
+		.opcode = MODE_SELECT_10,
+		.cdb_length = 10,
+		.cdb_usage = {MODE_SELECT_10, 0x11, 0x00, 0x00, 0x00, 0x00, 
+		              0x00, 0xff, 0xff, CONTROL_BYTE},
+	};
+
+	cdb_info opcode_receive_copy_results = {
+		.opcode = RECEIVE_COPY_RESULTS,
+		.cdb_length = 16,
+		.cdb_usage = {RECEIVE_COPY_RESULTS, 0x05, 0xff, 0xff, 0xff, 0xff,
+		              0x00, 0x00, 0x00, 0x00, 0xff, 0xff, 0xff, 0xff, 
+			      0x00, CONTROL_BYTE},     
+	};
+
+	cdb_info opcode_mi_report_target_pgs = {
+		.opcode = MAINTENANCE_IN,
+		.service_action = MI_REPORT_TARGET_PGS,
+		.cdb_length = 12,
+		.cdb_usage = {MAINTENANCE_IN, MI_REPORT_TARGET_PGS, 0x00, 0x00,
+			      0x00, 0x00, 0xff, 0xff, 0xff, 0xff, 0x00, 
+			      CONTROL_BYTE},
+	};
+
+	cdb_info opcode_mi_supported_operation_codes = {
+		.opcode = MAINTENANCE_IN,
+		.service_action = MI_REPORT_SUPPORTED_OPERATION_CODES,
+		.cdb_length = 12,
+		.cdb_usage = {MAINTENANCE_IN, MI_REPORT_SUPPORTED_OPERATION_CODES,
+			      0x87, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x00, 
+			      CONTROL_BYTE},
+	};
+	
+	cdb_info opcode_unmap = {
+		.opcode = UNMAP,
+		.cdb_length = 10,
+		.cdb_usage = {UNMAP, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0xff,
+		              0xff, CONTROL_BYTE},
+	};
+
+	cdb_info opcode_extended_copy = {
+		.opcode = EXTENDED_COPY,
+		.cdb_length = 16,
+		.cdb_usage = {EXTENDED_COPY, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+		              0x00, 0x00, 0xff, 0xff, 0xff, 0xff, 0x00, CONTROL_BYTE},
+	};
+	
+	cdb_info opcode_read_16 = {
+		.opcode = READ_16,
+		.cdb_length = 16,
+		.cdb_usage = {READ_16, 0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 
+		              0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x00, CONTROL_BYTE},
+	};
+
+	cdb_info opcode_write_16 = {
+		.opcode = WRITE_16,
+		.cdb_length = 16,
+		.cdb_usage = {WRITE_16, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+		              0x00, 0x00, 0xff, 0xff, 0xff, 0xff, CONTROL_BYTE},
+	};
+
+	cdb_info opcode_write_same_16 = {
+		.opcode = WRITE_SAME_16,
+		.cdb_length = 16,
+		.cdb_usage = {WRITE_SAME_16, 0xf8, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 
+		              0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x00, CONTROL_BYTE},
+	};
+
+	cdb_info opcode_write_verify_16 = {
+		.opcode = WRITE_VERIFY_16,
+		.cdb_length = 16,
+		.cdb_usage = {WRITE_VERIFY, 0xf2, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 
+		              0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x00, CONTROL_BYTE},
+	};
+
+	cdb_info opcode_format_unit = {
+		.opcode = FORMAT_UNIT,
+		.cdb_length = 6,
+		.cdb_usage = {FORMAT_UNIT, 0x00, 0xff, 0xff, 0xff, CONTROL_BYTE},
+	};
+
+	switch(op){
+		case INQUIRY:
```

Comment:
Should keep the code style the same as the current ones.
like:

    switch () {
    case XX:
            .....
            break;
    case YY:
    ....
    }